core/os/dir_access.cpp

```diff
@@ -170,7 +170,8 @@ Error DirAccess::make_dir_recursive(String p_dir) {
 		curpath = curpath.plus_file(subdirs[i]);
 		Error err = make_dir(curpath);
 		if (err != OK && err != ERR_ALREADY_EXISTS) {
-			ERR_FAIL_V(err);
+			String errStr = "Failed trying to access directory: " + curpath;
```

Comment:
Note that we use `snake_case` for identifiers, so this should be `err_str`.
But better, you don't need to make it a variable at all, just use:
```
ERR_FAIL_V_MSG(err, "Could not create directory: " + curpath);
```

Reply:
Fixed.